osdsctl/cli/volume.go

```diff
@@ -164,3 +173,25 @@ func volumeDeleteAction(cmd *cobra.Command, args []string) {
 	}
 	fmt.Printf("Delete volume(%s) success.\n", args[0])
 }
+
+func volumeUpdateAction(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		fmt.Println("The number of args is not correct!")
+		cmd.Usage()
+		os.Exit(1)
+	}
+
+	vol := &model.VolumeSpec{
+		Name:        volName,
+		Description: volDesp,
+	}
+
+	resp, err := client.UpdateVolume(args[0], vol)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
This error infomation should print to stderr.  You can use the fmt.Fprintln(os.Stderr, err) instead of it .